### DISCUSSION TITLE
appchannelTest.py Make error

### DISCUSSION THREAD

**Discussion by crazy-intern1** [bitcraze/discussions#1482] [Q&A]
I'm an intern doing my research with the crazyflie drones, I want to try basic P2P communication using radio dongle.

I want to test this code "appchannelTest.py" and I'm getting errors when I flash it with "make" 
the error message: 
#include "core_cm4.h" No such file or directory
the same for #include "FreeRTOS" 
I tried to flash it in cd crazyflie-firmware-experimnetal/examples/app_appchannel_test and 
cd crazyflie-firmware-experimental/app_api 
but getting same error message 

*I cloned this respiratory 'crazyflie-firmware-experimental'
but i think there is something missing or I'm not doing it right. 

Also, if there is any simpler code/project to understand/verify the P2P between the drones would be appreciated !

**Comment by knmcguire** [comment]
Hmm... it seems that your repository is missing some submodules. Have you cloned the repo with the '--recursive' flag? If not, you'll need to follow these instructions that hold for the regular firmware: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/building-and-flashing/build/#cloning

Have you worked with the crazyflie firmware before? It might be good to first get familiar with it and get it into a flying condition. The experimental repository is, as the name suggest, experimental demo work so I'm not sure if it would be the best option to get started with.

In the regular firmware, there are app examples available for peer 2 peer communication, so once you are ready you can take a look at it: https://github.com/bitcraze/crazyflie-firmware/tree/master/examples/app_peer_to_peer
